src/js/components/Notification/stories/typescript/Themed.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { Box, Grommet, Notification } from 'grommet';
+
+import { deepMerge } from 'grommet/utils';
+import { grommet, ThemeType } from 'grommet/themes';
+import { DisabledOutline, CircleQuestion } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customTheme: ThemeType = deepMerge(grommet, {
+  global: {
+    font: {
+      family: 'Arial',
+    },
+  },
+  notification: {
+    container: {
+      // any BoxProps
+      pad: { horizontal: 'large', vertical: '36px' },
```

Comment:
Not typical to use a non-tshirt size here -- any way we can make this a t-shirt size?